external/js/cothority/src/log.ts

```diff
@@ -137,40 +137,23 @@ export class Logger {
 
     catch(e: (Error | string), ...args: any) {
         let errMsg = e;
-        if ((e as Error).message) {
+        if (e instanceof Error) {
             errMsg = (e as Error).message;
```

Comment:
No need to cast now :)
```suggestion
             errMsg = e.message;
```